src/site/stages/build/process-cms-exports/transformers/node-person_profile.js

```diff
@@ -0,0 +1,38 @@
+const { getDrupalValue } = require('./helpers');
+
+const transform = entity => ({
+  entityType: 'node',
+  entityBundle: 'person_profile',
+  entityUrl: {
+    path: entity.path[0].alias,
+  },
+  fieldBody: getDrupalValue(entity.fieldBody),
+  fieldDescription: getDrupalValue(entity.fieldDescription),
+  fieldEmailAddress: getDrupalValue(entity.fieldEmailAddress),
+  fieldLastName: getDrupalValue(entity.fieldLastName),
+  fieldMedia: entity.fieldMedia.length > 0 ? entity.fieldMedia[0] : null,
+  fieldNameFirst: getDrupalValue(entity.fieldNameFirst),
+  fieldOffice: {
+    entity: {
+      entityLabel: entity.fieldOffice[0].entity.entityLabel,
+      entityType: entity.fieldOffice[0].entity.entityType,
```

Comment:
I considered destructuring these but then felt like it wouldn't help readability that much.  The `fieldOffice` transformer may also undergo some changes once we learn more about why it selectively appears in `pages.json`.